src/adhocracy_frontend/adhocracy_frontend/tests/acceptance/CommentSpec.js

```diff
@@ -13,22 +13,22 @@ describe("comments", function() {
     it("can be created", function() {
         var page = new EmbeddedCommentsPage("c1").get();
         var comment = page.createComment("comment 1");
-        expect(comment.isPresent()).toBe(true);
+        expect(comment.isDisplayed()).toBe(true);
```

Comment:
> It was needed to make the test pass after making the backend server work asynchron (increase workers from 1 to 2). I did not have time to investigate this.

maybe it was just because i did not call the promise function for the wait condition